src/components/SecondaryPanes/Expressions.js

```diff
@@ -78,18 +79,31 @@ class Expressions extends PureComponent<Props, State> {
     deleteExpression(expression);
   }
 
-  inputKeyPress(e, expression) {
+  async inputKeyPress(e, expression) {
+    const target = e.target;
+    target.className = "input-expression";
```

Comment:
Is this part needed? I think the reason it exists is to make sure we reset to the non-error state? I ask because it appears to already be set to this value by default [here](https://github.com/khal0988/debugger.html/blob/55c80e90e8c3b2b9ba18a20a23689e44222347b8/src/components/SecondaryPanes/Expressions.js#L114)